digitalidentity/service_test.go

```diff
@@ -29,7 +44,7 @@ func ExampleCreateShareSession() {
 		do: func(*http.Request) (*http.Response, error) {
 			return &http.Response{
 				StatusCode: 201,
-				Body:       io.NopCloser(strings.NewReader(`{"qrcode":"https://code.yoti.com/CAEaJDQzNzllZDc0LTU0YjItNDkxMy04OTE4LTExYzM2ZDU2OTU3ZDAC","ref_id":"0"}`)),
+				Body:       io.NopCloser(strings.NewReader(`{"status":"success","ref_id":"0"}`)),
```

Comment:
the response as described in https://lampkicking.atlassian.net/wiki/spaces/CON/pages/3756458012/Share+v2+Flows#API-Reference
should be:
{
  "id": "",
  "status": "",
  "expiry": ""
}